  Header header
  string source
  string destination
  string command
  nav_msgs/OccupancyGrid data